user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Apply_Data_Feed.md

```diff
@@ -41,6 +41,8 @@ To apply a data feed or change the data feed of a component:
 
    - A data feed can also be provided by a **feed component**. When such a component has been added to the dashboard or low-code app, the *Feeds* section is added to the available data in the *Data* pane. You can then drag an entry from this section to a component in order to link the component to the feed component.
 
+   - From DataMiner 10.3.0 [CU21]/10.4.0 [CU9]/10.4.12 onwards<!--RN 41039-->, data can also be provided by a **variable**. [Variables](xref:Variables) are reusable data objects that persist across sessions and can be dynamically updated.
```

Comment:
The "persist across sessions" might give the wrong impression that variables will keep their latest value when refreshing the app. Variables hold their initial value on app load, and can be overwritten in LCA through actions. They then hold that new value "in memory", which will be lost when you refresh the app again.

Reply:
```suggestion
   - From DataMiner 10.3.0 [CU21]/10.4.0 [CU9]/10.4.12 onwards<!--RN 41039-->, data can also be provided by a **variable**. [Variables](xref:Variables) are reusable data objects that can be dynamically updated.
```